pageobjects/newyoga.js

```diff
@@ -1,12 +1,8 @@
-let newyoga=function(){
-    let nextbutton=element.all(by.xpath("//body/div[2]/main[1]/div[3]/div[1]/div[4]/div[2]/ul[1]/li[4]/a[1]"));
+let newYoga = function(){
+    let nextButton = element.all(by.xpath("//body/div[2]/main[1]/div[3]/div[1]/div[4]/div[2]/ul[1]/li[4]/a[1]"));
```

Comment:
is this the only way to write this path?

Reply:
Yes